functests/4_latency/latency.go

```diff
@@ -459,7 +460,9 @@ func createLatencyTestPod(testPod *corev1.Pod, node *corev1.Node, logName string
 		testlog.Error(err)
 		logEventsForPod(testPod)
 	}
-	Expect(err).ToNot(HaveOccurred())
+	updatedPod := &corev1.Pod{}
+	_ = testclient.Client.Get(context.TODO(), client.ObjectKeyFromObject(testPod), updatedPod)
```

Comment:
do we really need an empty assignment here ?

Reply:
I don't see the point of checking that twice (already checked in  https://github.com/shajmakh/performance-addon-operators/blob/0f197be30938d8e25b7556eab7667349614f93e8/functests/utils/pods/pods.go#L114) and we surely don't want to fail the test for this as it is used only for reporting.

Reply:
I think the spirit of the comment was that you can just use
```
testclient.Client.Get(context.TODO(), client.ObjectKeyFromObject(testPod), updatedPod)
```
(provided the linter likes this way)

Reply:
oh i see, makes sense, will fix it